lib/std/os/windows.zig

```diff
@@ -152,6 +152,22 @@ pub fn OpenFile(sub_path_w: []const u16, options: OpenFileOptions) OpenError!HAN
     }
 }
 
+pub fn GetCurrentProcess() HANDLE {
+    return teb().ClientId.UniqueProcess;
+}
```

Comment:
This actually does not match the Windows implementation.

`GetCurrentProcess()` and `GetCurrentThread()` are documented to return constant pseudo-handles instead, currently `(HANDLE)-1` and `(HANDLE)-2`, respectively. I think we probably want to match this behavior (and then just inline the field access into the ID-retrieving functions).

Reply:
Oh yeah, I was wondering what those constants in ReactOS meant. Alternatively, I could just remove these since nobody in the stdlib is using them.